### PR TITLE
Add acceptance tests

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,6 +9,8 @@ gem 'rake'
 gem 'rspec', '~> 3.0'
 gem 'rspec-puppet', '~> 2.3'
 gem 'rspec-puppet-facts', '>= 1.7'
+gem 'beaker-rspec', {"groups"=>["system_tests"]}
```
Comment (domcleal) [inline Gemfile]: Missing from .sync.yml

Comment (ekohl) [inline Gemfile]: Let's get https://github.com/theforeman/foreman-installer-modulesync/pull/55 merged and then I'll base it on that.

**Comment by mmoll** [comment]
this needs a rebase now :)

**Comment by ekohl** [comment]
Rebased and I believe I addressed the comments.

**Comment by domcleal** [comment]
Merged, thanks @ekohl.

**Pull request merged**